vividus-plugin-rest-api/src/main/java/org/vividus/http/PublishingAttachmentInterceptor.java

```diff
@@ -73,27 +76,42 @@ public void process(HttpRequest request, EntityDetails entityDetails, HttpContex
                 {
                     LOGGER.error("Error is occurred at HTTP message parsing", e);
                 }
+                bodyForCurl = entity instanceof ByteArrayEntity
+                        ? "<binary_content>".getBytes(StandardCharsets.UTF_8) : body;
```

Comment:
that's not really correct, it will produce invalid curl command: we can't pass binary content as a string